package.json

```diff
@@ -41,7 +41,7 @@
         "start-server-and-test": "^1.14.0"
     },
     "dependencies": {
-        "@dhis2/app-runtime": "^3.2.6",
+        "@dhis2/app-runtime": "^3.3.0",
```

Comment:
Adds ability for mutations to `dataValues` and `dataValues/file` to use `multipart/form-data` body type